public/video-ui/src/components/Videos/VideoItem.js

```diff
@@ -10,7 +10,7 @@ export default class VideoItem extends React.Component {
 
             <div className="grid__info">
               <div className="grid__image">
-                <img src="http://placehold.it/350x150?text=image" alt="image"/>
+                <img src={this.props.video.posterUrl || ''} alt={this.props.video.title}/>
```

Comment:
Can we not use posterUrl (I don't think it exists anymore?) posterImage.master.file I think instead